src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -90,7 +90,8 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
              => new KeyToValueMappingEstimator(CatalogUtils.GetEnvironment(catalog), ColumnOptions.ConvertToValueTuples(columns));
 
         /// <summary>
-        /// Convert the key types back to their original vectors.
+        /// Maps key types (or key values) into a floating point vector suitable for learning (variously, one-hot-encoding,
```

Comment:
>variously [](start = 95, length = 9)

While I did use this language in the issue, it is not suitable language for documentation. The way in which I write is not generally suitable language for documentation.

Reply:
Oh come'on you are the [Shakespeare ](https://en.wikipedia.org/wiki/William_Shakespeare) of our team ... I'll change the language to be more ideal.